test/src/test/java/org/corfudb/AbstractCorfuTest.java

```diff
@@ -102,6 +225,49 @@ protected void succeeded(Description description) {
             }
             System.out.print(ansi().a("[").fg(Ansi.Color.GREEN).a("PASS")
                     .reset().a("]" + testStatus).newline());
+            System.out.flush();
+        }
+
+        /** Print a list of the active threads.
+         *
+         */
+        protected void printThreads() {
```

Comment:
do you need to suppress System.out.println warnings in code compile ?

Reply:
It only prints if there is a failure, so I assume you want to keep this in.